### ISSUE TITLE
Project Status Update

### ISSUE THREAD

**Issue by RoboDonut** [aws/chalice#726]
Hello, 

I'm posting bc I am rather curious about the status of Chalice. A few days ago it was brought up on gitter that commits have slowed and folks, including myself, are a bit concerned. We really love Chalice. It makes building APIs on AWS very easy. Speaking for myself I really want to keep using it and hope the project continues.

To that end, could you give us a little info on the project? Is it going to be supported and developed further? 

many thanks for any response and info you may give.

**Comment by jamesls** [comment]
Hi, Chalice is still being developed and supported.  While commits over the last month have been a bit slower than what we've done in the past, there's been ongoing internal refactorings to chalice that don't have any end user visibility just yet.

The main thing I've been working on now is the rewrite of the deployer (https://github.com/aws/chalice/issues/604) as we've realized that in order to implement some of the bigger feature requests (https://github.com/aws/chalice/issues/516, https://github.com/aws/chalice/issues/640, etc.) the way we do deployments needs to change.  Primarily, this is to remove the hard coded notion of "API Gateway + Lambda functions" as being a Chalice app.  This has the side effect of blocking some other feature work until it's done.  As a result, some of the smaller pull requests and issues haven't been getting the attention they should.

I think what I'll suggest to our team is that we allocate more time to staying on top of pull requests and smaller features in addition to the new deployer rewrite.

FWIW, we pushed out a patch release today that contains several enhancements and bug fixes (https://github.com/aws/chalice/blob/master/CHANGELOG.rst#111).

Thanks for the interest in the project!
